src/utils/Configuration.ts

```diff
@@ -93,6 +93,14 @@ class Configuration {
         return this.config.mot;
     }
 
+    /**
+     * Returns current version of Trailer Service
+     * @returns Trailer service version
+     */
+    public getTrailerServiceVersion(): string {
+        return process.env.TRAILERS_VERSION || "v1";
+    }
+
```

Comment:
Hmm, What does this give in practice? I figure if we did publish a new version of the API, V1 would still exist and moving to version two would mean a contract change - so we would need to do more than just bump the version no anyways.

Reply:
Contract changes in general shouldn't break the existing behaviour. In our case we are expecting a TrailerRegistration object. In either case `process.env.TRAILERS_VERSION` is an environment variable which we can control with any value and would default to V1 if no value is provided. So it won't automatically switch to a different version unless we change.

Reply:
I think it is best to throw an error if no version is given as the service would break anyway,
let's say the service is deployed to /v2 stage and no TRAILERS_VERSION given so it will default to /v1 and won't work anyway (or still use give some wrong information if we had to debug, we send /v2 payload but it will be read as /v1 in this service).
I would also think that we don't need to maintain this version in here as it should be given by the service itself  (trailers-registration) through the event in the handler.
We have to check with Tanio how the /v1 is done in the pipeline if we don't want to pass it from the trailers-project so we can mock it accordingly.

Or is this function to return the path and proxy for the handler to process? if it is the case it is probably best to rename as it is misleading and in fact we don't even seem to be using `TRAILERS_VERSION`, I can't see the variable being injected in our service ?